assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -266,11 +266,13 @@ storage:
           :FORWARD ACCEPT [0:0]
           :OUTPUT ACCEPT [0:0]
           -A INPUT -i lo -j ACCEPT
-          -A INPUT -p tcp --dport 22 -j ACCEPT
-          -A INPUT -p tcp --dport 179 -j ACCEPT
-          -A INPUT -p tcp --dport 2379 -j ACCEPT
-          -A INPUT -p tcp --dport 2380 -j ACCEPT
-          -A INPUT -p tcp --dport 2381 -j ACCEPT
+          %{ for management_cidr in management_cidrs ~}
+          -A INPUT -s ${management_cidr} -p tcp --dport 22 -j ACCEPT
```

Comment:
Grr, I'd say we disable ssh (and it will be open when calico starts) but bootkube will fail :-/

Reply:
We actually copy secrets over SSH to all controller nodes, not only to the bootstrap one :/

Reply:
Yeah, exactly :-/

Totally out of scope, but I'd like to play with adding those to ignition compressed (it seems ignition supports compression, not sure if packet will get in the way or if we are using something older, etc.)

Reply:
But yeah, nothing to do here with this, we need it for now :-/